include/MainWindow.h

```diff
@@ -224,6 +226,7 @@ public slots:
 
 	QBasicTimer m_updateTimer;
 	QTimer m_autoSaveTimer;
+	QTimer m_recordCountDownTimer;
```

Comment:
These timer facilities are not needed in the GUI, the model should handle the countdown.